moveit_ros/moveit_servo/include/moveit_servo/servo_calcs.h

```diff
@@ -198,51 +196,11 @@ class ServoCalcs
    */
   void insertRedundantPointsIntoTrajectory(trajectory_msgs::msg::JointTrajectory& joint_trajectory, int count) const;
```

Comment:
This might be another candidate for removal. @sea-bass when you run simulations in Gazebo, is this necessary?

Reply:
This is what is called with the `use_gazebo` parameter, right? In practice we haven't seen this make much of a difference, so it can probably go away too.

Reply:
Have taken it out as well.